libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -1,27 +1,29 @@
 <form [formGroup]="searchForm" (submit)="searchBooks()">
   <mat-form-field floatLabel="never">
-    <input
+    <label class="px-10">Search for books to add to your reading list</label>
```

Comment:
`px-10`? What does this class do? Where is it defined?

Reply:
Removed unused class(px-10).